frontend/vue.config.js

```diff
@@ -1,3 +1,28 @@
+const path = require('path')
+const PrerenderSpaPlugin = require('prerender-spa-plugin')
+
+const productionPlugins = [
+  new PrerenderSpaPlugin({
+    staticDir: path.join(__dirname, '../backend/public'),
+    routes: ['/', '/signup'],
+    renderer: new PrerenderSpaPlugin.PuppeteerRenderer({
+      // We need to inject a value so we're able to
+      // detect if the page is currently pre-rendered.
+      inject: {},
+      // Our view component is rendered after the API
+      // request has fetched all the necessary data,
+      // so we create a snapshot of the page after the
+      // `data-view` attribute exists in the DOM.
+      renderAfterElementExists: '[data-view]'
```

Comment:
oh..kay
So it renders... and after the element with `data-view` exists, it creates a snapshot... as opposed to...?

Reply:
Also, some pages which aren't in the list of routes have `data-view` e.g. SubjectList - does this prerender those too?

Reply:
Only if I specified. I played around but SubjectList is a more complicated route.